### PR TITLE
Execution Plan Telemetry for Plan Comparisons

### PR DESCRIPTION
**Pull request by lewis-sanchez**
This PR adds telemetry to execution plans. Specific examples include how plans are invoked, when the execution plan button is used, and when the plan comparison properties button is used.

### CODE REVIEW

**Review by aasimkhan30 — APPROVED** [review]
Please wait for Charles to approve the telemetry events before merging

File: src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts:538]: You can just call `sendActionEvent` directly if you aren't adding custom properties

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**
